lib/sidekiq/instrument/mixin.rb

```diff
@@ -12,6 +12,13 @@ def worker_dog_options(worker)
       { tags: ["queue:#{queue_name(worker)}", "worker:#{underscore(class_name(worker))}"] }
     end
 
+    def max_retries(worker)
```

Comment:
I think I would prefer:

```
    def max_retries2(worker)
      retries = worker.class.get_sidekiq_options['retry'] || Sidekiq[:max_retries]
      return Sidekiq[:max_retries] if retries.to_s.eql?("true")
      return 0 if retries.to_s.eql?("false")
      retries
    end
```

Because using the same variable with multiple reassignments always feels dangerous to me, but YMMV.

Reply:
I agree - I think it's cleaner. I'll make the change!